dags/bqetl_kpis_shredder.py

```diff
@@ -121,20 +121,6 @@
     fenix_active_users_aggregates_for_deletion_requests.set_upstream(
         wait_for_clients_last_seen_joined
     )
-    wait_for_copy_deduplicate_all = ExternalTaskSensor(
-        task_id="wait_for_copy_deduplicate_all",
-        external_dag_id="copy_deduplicate",
-        external_task_id="copy_deduplicate_all",
-        check_existence=True,
-        mode="reschedule",
-        allowed_states=ALLOWED_STATES,
-        failed_states=FAILED_STATES,
-        pool="DATA_ENG_EXTERNALTASKSENSOR",
-    )
-
-    fenix_active_users_aggregates_for_deletion_requests.set_upstream(
-        wait_for_copy_deduplicate_all
-    )
```

Comment:
Why did these get removed?

Reply:
I might have missed them in the sql generation step. Running `./bqetl generate all` now to see if they come back

Reply:
and back they are